apps/eslint/index.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
             (item.type === 'ExportDefaultDeclaration' &&
               item.declaration.type === 'ClassDeclaration') ||
             (item.type === 'ExportNamedDeclaration' &&
-              item.declaration?.type === 'ClassDeclaration')
+              item.declaration &&
```

Comment:
I was getting some linting errors related to this.. just going to clean that up so it doesn't happen to anyone else.